village/src/App.js

```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
-
+import axios from 'axios';
+import {BrowserRouter as Router, Route, NavLink } from "react-router-dom";
```

Comment:
No need for `BrowserRouter` here. It's only needed in `index.js`
```suggestion
import {Route, NavLink } from "react-router-dom";
```